.travis/install.sh

```diff
@@ -15,6 +15,7 @@ pip install -U pytest
 pip install -U pytest-cov
 pip install pyroma
 pip install test-image-results
+pip install numpy
```

Comment:
Is it the best place for such requirements? Do I need to add it to docker images?

Reply:
install.sh is used by the Travis CI "native" build jobs. It's not used by the Docker jobs.

So install.sh as a minimum, and it's good to also include it in the Docker images for wider OS test coverage.

Reply:
Send pull request https://github.com/python-pillow/docker-images/pull/25